src/js/test/supervisors/FileManager.test.ts

```diff
@@ -275,4 +275,40 @@ describe("FileManager", () => {
       done();
     });
   });
+
+  it(
+    "should remove a coprocessor, if it's removed from active folder and " +
+      "memory, although disable_coproc request fails",
+    function (done) {
+      const handle = createHandle({
+        globalId: hash64(Buffer.from("file"), 0).readBigUInt64LE(),
+      });
+      const {
+        getCoprocessor,
+        moveCoprocessor,
+        disableCoprocessor,
+      } = createStubs(sinonInstance);
+      moveCoprocessor.returns(Promise.resolve(handle));
+      getCoprocessor.returns(Promise.resolve(handle));
+      disableCoprocessor.reset();
```

Comment:
Why do you need to reset only this one?

Reply:
Because `disableCoprocessor` has a default stub value, for another test, I reset that stub and add new behavior

Reply:
Is it just one test?

Reply:
Yes, for any other test it returns a `Promise.resolve()` for emulating a server response, but in this case, I need to emulate a fail response, for this reason, I reset its behaviors